torch/_dynamo/eval_frame.py

```diff
@@ -464,6 +476,18 @@ def _fn(*args, **kwargs):
             try:
                 return fn(*args, **kwargs)
             finally:
+                # Log all the traced frames.
+                code = fn.__code__
+                msg = f"starting from {code.co_name} {code.co_filename}:{code.co_firstlineno}"
+                msg += ", torchdynamo attempted to trace the following frames: ["
+                if len(dynamo_tls.traced_frame_infos) > 0:
+                    msg += "\n"
+                    for traced_frame_info in dynamo_tls.traced_frame_infos:
+                        msg += "  * " + traced_frame_info + "\n"
+                msg += "]"
+                dynamo_tls.traced_frame_infos.clear()
+                log.info(msg)
```

Comment:
@williamwen42 Looks like this code triggers tracing as well, specifically `CatchErrorsWrapper.__call__`. Are you aware of any easy way to skip tracing for this logging portion?

Reply:
Figured it out, just need to run this after `_maybe_set_eval_frame` right below.